Library/Formula/bro.rb

```diff
@@ -22,6 +22,12 @@ class Bro < Formula
   depends_on "geoip" => :recommended
   depends_on "openssl"
 
+  # Initially empty directories
+  skip_clean "spool"
+  skip_clean "logs"
```

Comment:
`spool` and `logs` would typically belong in `var` so they wouldn't be removed anyhow. Where are they stored now?